fabric-object-builders-v0/src/main/java/net/fabricmc/fabric/api/entity/FabricEntityTypeBuilder.java

```diff
@@ -81,25 +72,42 @@ protected FabricEntityTypeBuilder(EntityCategory category, EntityType.EntityFact
 		return this;
 	}
 
+	/**
+	 * @deprecated Old name that will be removed in a later version. Use {@link FabricEntityTypeBuilder#makeFireImmune()}.
+	 */
+	@Deprecated
 	public FabricEntityTypeBuilder<T> setImmuneToFire() {
-		this.immuneToFire = true;
+		return makeFireImmune();
+	}
+
+	public FabricEntityTypeBuilder<T> makeFireImmune() {
+		this.fireImmune = true;
+		return this;
+	}
+
+	public FabricEntityTypeBuilder<T> spawnableFarFromPlayer() {
+		this.spawnableFarFromPlayer = true;
 		return this;
 	}
 
 	/**
-	 * @deprecated Use {@link FabricEntityTypeBuilder#size(EntityDimensions)}
+	 * @deprecated Old name that will be removed in a later version. Use {@link FabricEntityTypeBuilder#setDimensions(EntityDimensions size)}.
 	 */
 	@Deprecated
-	public FabricEntityTypeBuilder<T> size(float width, float height) {
-		this.size = EntityDimensions.changing(width, height);
-		return this;
+	public FabricEntityTypeBuilder<T> size(EntityDimensions size) {
+		return setDimensions(size);
 	}
 
-	public FabricEntityTypeBuilder<T> size(EntityDimensions size) {
+	public FabricEntityTypeBuilder<T> setDimensions(EntityDimensions size) {
 		this.size = size;
 		return this;
 	}
 
+	public FabricEntityTypeBuilder<T> setMaxDespawnDistance(int maxDespawnDistance) {
```

Comment:
```suggestion
	public FabricEntityTypeBuilder<T> maxDespawnDistance(int maxDespawnDistance) {
```

Reply:
Most of these method name changes liach as been going through the review are actual java convention, since builders do not usually have a `set` prefix on their methods. So I would advise following those.